Sources/Extensions/Foundation/DateExtensions.swift

```diff
@@ -952,3 +952,15 @@ public extension Date {
 	}
 	
 }
+
+extension Date: ExpressibleByIntegerLiteral {
```

Comment:
Please mark this as `public`

Reply:
@omaralbeik , we can't make it public when using protocol conformance. I've made it public, by removing protocol conformance